src/Symfony/Component/Serializer/Mapping/ClassDiscriminatorResolverInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Mapping;
+
+/**
+ * Knows how to get the class discriminator mapping for classes and objects.
+ *
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+interface ClassDiscriminatorResolverInterface
+{
+    /**
+     * @param string $class
+     *
+     * @return ClassDiscriminatorMapping|null
+     */
+    public function getMappingForClass(string $class);
```

Comment:
A return type can be specified for each method

Reply:
We are targeting PHP 7.0, not 7.1, right? Can't add the return type when it can return `null` then 🤔

Reply:
Symfony 4 supports PHP >= 7.1.3